modules/custom/az_person/config/install/core.entity_view_display.node.az_person.default.yml

```diff
@@ -19,11 +19,211 @@ dependencies:
+        speed: fast
+    group_col_12_image:
+      children:
+        - group_card
+      label: 'Col 12 - Image'
+      parent_name: group_row
+      region: content
+      weight: 4
+      format_type: html_element
+      format_settings:
+        classes: 'col-12 col-md-4 float-md-left'
+        show_empty_fields: false
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_col_bio:
+      children:
+        - field_az_body
+        - field_az_degrees
+        - field_az_person_category
+        - field_az_person_category_sec
+      label: 'Col - Bio'
+      parent_name: group_row
+      region: content
+      weight: 6
+      format_type: html_element
+      format_settings:
+        classes: 'col-12 col-md-8 float-md-right'
+        show_empty_fields: false
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_card:
+      children:
+        - field_az_media_image
+        - group_card_body
+      label: Card
+      parent_name: group_col_12_image
+      region: content
+      weight: 5
+      format_type: html_element
+      format_settings:
+        classes: 'card mb-5'
+        show_empty_fields: false
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_card_body:
+      children:
+        - field_az_address
+        - field_az_phones
+        - field_az_email
+        - field_az_pronouns
+        - field_az_links
+        - group_text_break
+      label: 'Card Body'
+      parent_name: group_card
+      region: content
+      weight: 7
+      format_type: html_element
+      format_settings:
+        classes: card-body
+        show_empty_fields: false
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_titles:
+      children:
+        - field_az_titles
+      label: Titles
+      parent_name: group_col_12_name_titles
+      region: content
+      weight: 8
+      format_type: html_element
+      format_settings:
+        classes: 'h4 d-block mt-0'
+        show_empty_fields: false
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_text_break:
+      children:
+        - field_az_attachments
+      label: 'Text Break'
+      parent_name: group_card_body
+      region: content
+      weight: 12
+      format_type: html_element
+      format_settings:
+        classes: text-break
```

Comment:
Final nitpick! Currently, on the default display there is vertical space between the person's Link(s) and their CV/Documents:

<img width="379" alt="image" src="https://user-images.githubusercontent.com/74572157/205367879-d81d6c49-c1c1-477c-a497-35dd1089c5cf.png">

Right now, this vertical space is not included in this PR:

<img width="376" alt="image" src="https://user-images.githubusercontent.com/74572157/205368040-ca98ad1a-a11d-4a34-8cb5-2197a35fb21c.png">

I'd recommend adding a bootstrap margin value of 3, to match the spacing between the address block and phone number/email block. The easiest change is to add `mt-3` to the Text Break HTML element, as suggested below. Otherwise, if we want to stick to margin-bottom classes when possible, perhaps a new wrapping div with the `mb-3` class could be added that would contain the four fields from Phone Number(s) to Link(s).

```suggestion
        classes: 'text-break mt-3'
```